server/utils/constants.js

```diff
@@ -310,46 +310,47 @@ const INTENTS = {
 };
 
 const DEVICE_FEATURE_CATEGORIES = {
-  LIGHT: 'light',
```

Comment:
Did you just sort them?

Reply:
Yes https://github.com/VonOx/Gladys/blob/aqi/server/utils/constants.js#L332